parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -55,6 +57,16 @@
  */
 public class ParquetProperties {
```

Comment:
The name is a little outdated now that you can pass an allocator.
ParquetConfiguration?